README.md

```diff
@@ -2,6 +2,7 @@
 # EOSIO - The Most Powerful Infrastructure for Decentralized Applications
 
 [![Build status](https://badge.buildkite.com/370fe5c79410f7d695e4e34c500b4e86e3ac021c6b1f739e20.svg?branch=master)](https://buildkite.com/EOSIO/eosio)
+[![Tag](https://img.shields.io/github/v/tag/eosio/eos.svg)](https://github.com/eosio/eos/releases)
```

Comment:
Wouldn't release be a better choice than tag? https://img.shields.io/github/v/release/eosio/eos.svg

Otherwise I think prerelease tags will get picked

Reply:
Good suggestion. already done.